test/passes/remove-unused-brs_enable-multivalue.txt

```diff
@@ -19,10 +19,11 @@
  )
  (func $b1 (param $i1 i32)
   (block $topmost
-   (block $block
+   (block
     (drop
      (i32.const 0)
     )
+    (nop)
```

Comment:
Do you know where these extra `(nop)`s are coming from?

Reply:
Good point, those shouldn't be there. This looks like a minor issue uncovered by this PR. It's not a regression on real-world code since this PR just affects text inputs, so I'll land this, but I'll fix it as a followup.